Flat/src/ModulesJets.cc

```diff
@@ -70,7 +70,7 @@ void PandaAnalyzer::JetBasics()
     }
 
 
-    if (jet.pt()>30) { // nominal jets
+    if (jet.pt()>20) { // nominal jets
```

Comment:
No. We cannot mess up other analyses like this. Make it configurable and only for h->bb.

Reply:
Oops, makes sense! ( We also use 30 GeV for ZH)